README.md

```diff
@@ -13,7 +13,7 @@ React
 
 ## Getting started
 To run the code, clone the repo using the command git clone https://github.com/imubarak234/Math-Magicians when the clone is complete you can cd to the file and open it up in your text editor. For more information on pulls and push on git checkout this link: https://guides.github.com/introduction/git-handbook/
-Run command npm init -y and npm start to run the applications 
+Run command `npm install` , `npm init -y` and `npm start` to run the applications 
```

Comment:
- [ ] In your README as the previous reviewer said no need for `npm init -y` to run the app. Because it does not install packages. Kindly remove it, please!